alexanderlopez/capstone_project
# Map unit tests

Add Mocha unit tests to regex function in threadHandler.js.

TODO: figure out how to run the tests without copying the function into the test file.

Test groups:
1. Check valid characters (uppercase, lowercase, numbers, dash, underscore) are accepted by the regex test
2. Check invalid character (everything else) are rejected by the regex test
3. Check randomly generate combinations of both groups of characters.
